src/task_types/resmoke_config_writer.rs

```diff
@@ -39,8 +36,6 @@ struct WriteConfigActorImpl {
     /// Test discovery service.
     test_discovery: Arc<dyn TestDiscovery>,
 
-    multiversion_service: Arc<dyn MultiversionService>,
```

Comment:
This code no longer cares about multiversion. Any multiversion information it needs will be included in the data passed into it.